docs/api/spec/components/parameters/base.yaml

```diff
@@ -29,3 +29,11 @@ logic_operator:
   schema:
     type: string
   description: The logic operator to use for the search query. Can either be all or any.
+  
+local:
+  in: query
+  name: local
+  required: false
```

Comment:
I think it would be safe to say it default to 0 here since when its omitted, the value in the service is an empty string and the code matches local == "1"

Reply:
You mean that it should default to 1, since 'default' has to be implicitly set to '0' for it to chain the req to the cluster?

Reply:
I think it should be define to 0 since the code will detect if there ar other members in the cluster.